<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set configuration</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDriverType">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Main Image File / File for Data Sim</string>
   </property>
   <widget class="QLineEdit" name="lineEditImageFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFileImg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAddFw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Config file for receiver (hex)</string>
   </property>
   <widget class="QLineEdit" name="lineEditAdditionalImageFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHaveSubs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFileAddFw">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAnalog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBoxBoardType">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPreDef">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Predefined config for board</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHaveAtts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Have atts</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGPSInterFreq">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditADCSampleRate">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFreqInterGPS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Intermediate freq, Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSampleRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample rate, Hz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHaveDbg">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Have debug ouput</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelDriver">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Driver type</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto start streams</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hacked data len</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxHackedLen">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxBit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>FPGA bitfile (bit)</string>
   </property>
   <widget class="QLineEdit" name="lineEditBitFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFileBit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
